Tribler/Test/Community/Triblerchain/test_community.py

```diff
@@ -353,3 +357,32 @@ def test_get_statistics_for_not_self(self):
         statistics = node.community.get_statistics(public_key=other.community.my_member.public_key)
         assert isinstance(statistics, dict), type(statistics)
         assert len(statistics) > 0
+
+    def test_get_trust(self):
+        """
+        Test that the trust nodes have for each other is the upload + the download total of all blocks.
+        """
+        # Arrange
+        node, other = self.create_nodes(2)
+        transaction = {'up': 10, 'down': 5, 'total_up': 10, 'total_down': 5}
+        TestTriblerChainCommunity.create_block(node, other, self._create_target(node, other), transaction)
+        TestTriblerChainCommunity.create_block(other, node, self._create_target(other, node), transaction)
+
+        # Get statistics
+        node_trust = blockingCallFromThread(reactor, node.community.get_trust, other.community.my_member)
```

Comment:
Why not simply decorate this whole method with an `@blockingcallonreactorthread` (and use `yield self.create_nodes(2)`)?